trainingportal/static/lessons/redTeam/red_ch1.md

```diff
@@ -3,17 +3,60 @@ Use nmap to scan for open ports of the provided host, identify applications runn
 #### Getting setup
 Here are instructions on how to get your pen-tester environment setup:
 
+#### Instructions for Completing the Challenge
+- Run nmap
+- Run metasploit or a different exploit script to take advantage of a vulnerability in the first identified app
+- Execute commands remotely
+- Find the flag program in the current directory and use it to sign the challenge salt: `./flag <SALT>`
+
 ##### Nmap
-Install nmap.
+Install nmap. You can use a unofficial docker image or install it directly on your VM: 
+
+`docker pull securecodebox/nmap`
+
+OR for Debian/Ubuntu:
+
+`apt-get install nmap`
+ 
+OR for RedHat/CentOS:
+
+`rpm -vhU https://nmap.org/dist/nmap-7.80-1.x86_64.rpm`
 
 Once nmap is installed you can scan the target host with `nmap <target_host> -Pn -p-`
 
 ##### Metasploit
-If you have Docker installed you can simply run the metasploit container image.
+If you have Docker installed you can simply run the metasploit container image. Here's how to do it:
 
+###### Pull metasploit docker image from DockerHub
 
-#### Instructions for Completing the Challenge
-- Run nmap
-- Run metasploit or a different exploit script to take advantage of a vulnerability in the first identified app
-- Execute commands remotely
-- Find the flag program in the current directory and use it to sign the challenge salt: `./flag <SALT>`
\ No newline at end of file
+`docker pull metasploitframework/metasploit-framework`
```

Comment:
Should we advise that metasploit is installed on a EC2 that is internet accessible to allow for the meterpreter to run?

Reply:
Sounds good. Added message before the docker pull.